README.md

```diff
@@ -4,6 +4,34 @@ A list of climate friendly extensions that can help you reduce the carbon footpr
 ## CO2 emissions trackers and green website raters
 
 ### Globemallow
+#### Description
+
+Use Globemallow to quickly view a sustainability grade and sustainability metrics for a website that you visit.
+Globemallow was created to help designers and developers build websites using more sustainable development practices.
+
+**The Sustainability Grade**
+Globemallow uses the US Academic Grading System:
+- A+ is a good grade for a sustainable website 
+- F will be the rating score for websites that could use more work to be more sustainable. 
+
+**Built on Open Source:**
+- Energy consumption estimations are supplied by The Shift Project: https://github.com/carbonalyser/Carbonalyser
+
+#### Browser compatibility
+Available on Chrome and Edge.
+
+#### Extension URL
+
+- Chrome: https://chrome.google.com/webstore/detail/globemallow/jibhiolaefbcfiahgolfpmngjefngdmd
+- Edge: https://microsoftedge.microsoft.com/addons/detail/globemallow/nbdlhghpkjhadffcigbopbbpdflcgnhc
+
+#### Extension size
+
+97.55KiB
+
+#### Extension last updated 
+
+4 September 2022
 
```

Comment:
Here I would add another linebreak so it's visually clearer a new app begins here.